routes/linkUrls.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const cache = require('memory-cache');
+const asyncHandler = require('express-async-handler');
+
+const handleCache = require('../helpers/handleCache');
+const commonContent = require('../helpers/commonContent');
+
+router.get('/:codenames', asyncHandler(async (req, res, next) => {
+    const codenames = req.params.codenames.split('/');
+
+    if (codenames.length === 0) {
+        return next();
+    } else {
```

Comment:
You don't need "else" here, since there is a return statement - the approach I describe reduces nesting…

Reply:
I can attend this while integrating Blob Provider into the web.